changelogs/current.yaml

```diff
@@ -10,6 +10,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: geoip
+  change: |
+    The lookup for asn information is fetch from ``asn_db`` if asn_db_path is defined and from ``isb_db`` if is asn_db_path is not defined.
```

Comment:
this should also go either to api docs or to user docs for maxmind provider. Since users will not be reading changelogs to learn how geolocation lookups work.